components/map/MapPin.tsx

```diff
@@ -0,0 +1,41 @@
+import { View } from 'react-native'
+
+import MapPinSvg from '@/assets/ui-icons/map-pin.svg'
```

Comment:
We import svgs as components from barrel files, so they always have same name, and it's easier to work with. This is  just a coding style decision :) 

e.g.
`import { MapPinSvg } from '@/assets/map`